scrapers.js

```diff
@@ -1686,6 +1686,30 @@ let scrapers = [
       };
     }
   },
+  {
+    country: 'AUS',
+    url: 'https://www.health.gov.au/news/health-alerts/novel-coronavirus-2019-ncov-health-alert/coronavirus-covid-19-current-situation-and-case-numbers',
```

Comment:
This little number makes your data authoritative vs other sources for AUS and its states, as well as provides the type so our forthcoming rating system #82 can rank the data source.

```suggestion
    url: 'https://www.health.gov.au/news/health-alerts/novel-coronavirus-2019-ncov-health-alert/coronavirus-covid-19-current-situation-and-case-numbers',
    type: 'table',
    priority: 1,
```